syft/frameworks/torch/functions.py

```diff
@@ -0,0 +1,5 @@
+from .tensors.interpreters import MultiPointerTensor
+
+
+def unite_pointers(*pointers):
```

Comment:
Combine pointers?

Reply:
are you recommending a name change or asking what teh functinoality is?

Reply:
Recommending a name change. I'm kinda either way on it though

Reply:
i like combine - will change